src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisProfilerEntryInstrumenter.cs

```diff
@@ -88,30 +95,37 @@ static bool GetCommandAndKey(
             StackExchangeRedisConnectionInstrumentation.CreationTags,
             startTime: command.CommandCreated);
 
-        if (activity == null)
+        if (activity is null && metrics.Enabled is false)
         {
             return null;
         }
 
-        activity.SetEndTime(command.CommandCreated + command.ElapsedTime);
+        activity?.SetEndTime(command.CommandCreated + command.ElapsedTime);
+        var meterTags = metrics.Enabled ?
+            new TagList(StackExchangeRedisConnectionInstrumentation.CreationTags) :
```

Comment:
creationtags is no longer needed, as it is already covered via Meter Tags.

Reply:
since you suggest removing the meter tags https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1982#discussion_r1725675861, I will keep it